test/functional/clusteradm/clusteradmE2E_test.go

```diff
@@ -0,0 +1,252 @@
+// Copyright Contributors to the Open Cluster Management project
+package clusteradme2e
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"open-cluster-management.io/clusteradm/test/functional/util"
+)
+
+var e2e, e2e2, e2e3 *util.TestE2eConfig
```

Comment:
it is better to define the var in each case

Reply:
ok